mk/distribution.mk

```diff
@@ -75,7 +75,7 @@ publish/pulp/$(DISTRIBUTION_TARGET_NAME)-$(1)-$(2):
 		-e USE_CLOUDSMITH=x \
 		-e USE_PULP=x \
 		-v $(TOP)/build/distributions/out:/files:ro \
-		-it \
+		-t \
```

Comment:
Is it enough to remove `-i`? That just controls stdin IIUC

Reply:
The error is: "the input device is not a TTY" so isn't that what this problem is?